homeassistant/components/mysensors/__init__.py

```diff
@@ -11,13 +11,15 @@
 
 from ... import config_entries
 from ...config_entries import ConfigEntry
+from ...const import CONF_OPTIMISTIC
```

Comment:
Please use absolute imports for the non mysensors integration package imports. Intra package imports should still be relative.